tests/zdalne_response_tests.py

```diff
@@ -0,0 +1,17 @@
+from source.response_utils.get_text import get_text
+
+ZDALNE_ANSWER = "Nie ma żadnych zdalnych, zdalne wymyśliliście sobie Wy, studenci."
```

Comment:
why polish word as a variable name?

Reply:
this is function name, not polish verb in this case